examples/basic_candlepin.pp

```diff
@@ -58,4 +58,5 @@
   truststore_password => $truststore_password,
   java_package        => 'java-11-openjdk',
   java_home           => '/usr/lib/jvm/jre-11',
+  artemis_client_dn   => Deferred('pick', ['', 'CN=ActiveMQ Artemis Deferred, OU=Artemis, O=ActiveMQ, L=AMQ, ST=AMQ, C=AMQ']),
```

Comment:
Why the empty string?

Reply:
I just wanted a function to be able to test passing a Deferred and pick seemed like a good one. I suppose I could just pass a single string to it instead of passing both an empty and a non-empty. I was thinking to make it more "realistic".